tests/src/common/TestUtils.java

```diff
@@ -138,6 +136,18 @@ public static JsonObject getVCAPServices() {
     }
 
     /**
+     * Gets a VCAP_SERVICES credentials as the json objects.
+     *
+     * @return VCAP credentials as a json object.
+     */
+    public static JsonObject getCredentials(String vcapService) {
+    	JsonArray vcapArray = getVCAPServices().get(vcapService).getAsJsonArray();
```

Comment:
please indent correctly.

Are we really willing to keep GSON in our repository? Why not spray.json?